core/templates/pages/learner-dashboard-page/community-lessons-tab.component.html

```diff
@@ -290,15 +290,15 @@
              container="body"
              placement="top">
           </i>
-          <oppia-exploration-summary-tile [getExplorationId]="summaryTile.id"
-                                          [getExplorationTitle]="summaryTile.title"
-                                          [getLastUpdatedMsec]="summaryTile.lastUpdatedMsec"
-                                          [getObjective]="summaryTile.objective"
-                                          [getCategory]="summaryTile.category"
-                                          [getRatings]="summaryTile.ratings"
-                                          [getNumViews]="summaryTile.numViews"
-                                          [getThumbnailIconUrl]="summaryTile.thumbnailIconUrl"
-                                          [getThumbnailBgColor]="summaryTile.thumbnailBgColor">
+          <oppia-exploration-summary-tile [explorationId]="summaryTile.id"
```

Comment:
Are these expected changes?

Reply:
@nithusha21  We're following this pattern of removing `get*` during migration from @Input properties and convert them to nouns instead. It was not covered during migration [here](https://github.com/oppia/oppia/pull/12056/files#diff-0bde4b671455e212d03d422d081c420ea3692bc7fb5674b0385410dc79cef9f4R4), but external evaluator requested the changes to do it here.